cmd/snap-preseed/preseed_linux.go

```diff
@@ -235,10 +235,11 @@ func prepareChroot(preseedChroot string) (*targetSnapdInfo, func(), error) {
 	}
 
 	fstype, fsopts := squashfs.FsType()
-	cmd := exec.Command("mount", "-t", fstype, "-o", strings.Join(fsopts, ","), coreSnapPath, where)
-	if err := cmd.Run(); err != nil {
+	mountArgs := []string{"-t", fstype, "-o", strings.Join(fsopts, ","), coreSnapPath, where}
+	cmd := exec.Command("mount", mountArgs...)
+	if out, err := cmd.CombinedOutput(); err != nil {
 		removeMountpoint()
-		return nil, nil, fmt.Errorf("cannot mount %s at %s in preseed mode: %v ", coreSnapPath, where, err)
+		return nil, nil, fmt.Errorf("cannot mount %s at %s in preseed mode: %v\n'mount %s' failed with: %s", coreSnapPath, where, err, strings.Join(mountArgs, " "), out)
```

Comment:
I was wondering if we should avoid the strings.Join() and just use %q instead of %s but after some more thinking I feel it's fine as is.